### DISCUSSION TITLE
请教一下配置好了目录，如何开始自动整理？目录里有以前就存在的文件，还有能不能整理完成后过一星期自动删除被硬链接的文件

### DISCUSSION THREAD

**Discussion by sheiy** [jxxghp/MoviePilot#3098] [问题咨询]
还有如何手动触发整理一次？
<img width="1223" alt="image" src="https://github.com/user-attachments/assets/f4439c5d-def1-4d12-ae11-60e24fac70c1">

**Comment by slashyue** [comment]
第一次整理需要打开文件管理，手动开始整理队列
